modules/recommended-configurable-storage-technology.adoc

```diff
@@ -84,39 +84,45 @@ In a non-scaled/high-availability (HA) {product-title} registry cluster deployme
 
 * The preferred storage technology is object storage followed by block storage. The
 storage technology does not have to support RWX access mode.
-* The storage technology must ensure read-after-write consistency. All NAS storage are not
-recommended for {product-title} Registry cluster deployment with production workloads.
-* While `hostPath` volumes are configurable for a non-scaled/HA {product-title} Registry, they are not recommended for cluster deployment.
+* The storage technology must ensure read-after-write consistency. File storage is
+not recommended for {product-title} Registry cluster deployment with production
+workloads.
+* While `hostPath` volumes are configurable for a non-scaled/HA {product-title}
+Registry, they are not recommended for cluster deployment.
 
 === Scaled registry
 
 In a scaled/HA {product-title} registry cluster deployment:
 
-* The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
-* File storage and block storage are not recommended for a scaled/HA {product-title} registry cluster deployment with production workloads.
+* The preferred storage technology is object storage.
+* Object storage must be S3 compliant.
```

Comment:
We support S3, GCS, Azure Blob Storage, Swift. It's strange to ask Azure Blob Storage to be S3 compliant. If you want to use S3 compliant storage, it should be S3 compliant (obviously).

Reply:
+1, list of supported backends (or a link to the list) would be great.